src/main/cpp/t02_new.cpp

```diff
@@ -25,10 +25,24 @@
 
 #include "t02_new.h"
 #include <iostream>
-
+#include <set>
 
 using namespace std;
 
 int t02_new() {
-    
+    int N;
+    cin >> N;
+    set <int> nums;
+    int c, a1, a2;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The scope of the variable 'a2' can be reduced.](https://app.codacy.com/manual/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=5091529)